build.gradle.kts

```diff
@@ -71,107 +73,208 @@ tasks.jar.configure {
   }
 }
 
-tasks.withType<JavaCompile>().configureEach { 
-    options.compilerArgs = options.compilerArgs + listOf(
-      "-Xlint:all"
-    )
+tasks.withType<JavaCompile>().configureEach {
+  options.compilerArgs.add("-Xlint:all")
 }
 
-/* TESTING */
-tasks.test.configure {
+// <editor-fold defaultstate="expanded" desc="Plugin Configuration">
+gradlePlugin {
+  plugins {
+    create("appengine") {
+      id = "$group.appengine"
+      implementationClass = "com.google.cloud.tools.gradle.appengine.AppEnginePlugin"
+      displayName = "App Engine Gradle Plugin"
+      description =
+        "This Gradle plugin provides tasks to build and deploy Google App Engine applications."
+    }
+    create("appengine-appenginewebxml") {
+      id = "$group.appengine-appenginewebxml"
+      implementationClass =
+        "com.google.cloud.tools.gradle.appengine.standard.AppEngineStandardPlugin"
+      displayName = "App Engine Gradle Plugin"
+      description =
+        "This Gradle plugin provides tasks to build and deploy Google App Engine applications."
+    }
+    create("appengine-appyaml") {
+      id = "$group.appengine-appyaml"
+      implementationClass =
+        "com.google.cloud.tools.gradle.appengine.appyaml.AppEngineAppYamlPlugin"
+      displayName = "App Engine Gradle Plugin"
+      description =
+        "This Gradle plugin provides tasks to build and deploy Google App Engine applications."
+    }
+    create("appengine-flexible") {
+      id = "$group.appengine-flexible"
+      implementationClass =
+        "com.google.cloud.tools.gradle.appengine.appyaml.AppEngineAppYamlPlugin"
+      displayName = "App Engine Gradle Plugin"
+      description =
+        "This Gradle plugin provides tasks to build and deploy Google App Engine applications."
+    }
+    create("appengine-standard") {
+      id = "$group.appengine-standard"
+      implementationClass =
+        "com.google.cloud.tools.gradle.appengine.standard.AppEngineStandardPlugin"
+      displayName = "App Engine Gradle Plugin"
+      description =
+        "This Gradle plugin provides tasks to build and deploy Google App Engine applications."
```

Comment:
I think I copied the values from the old properties. Of course, adding better descriptions would be nice, however, it would increase the PR surface for no reason